oneShow.js

```diff
@@ -0,0 +1,246 @@
+//You can edit ALL of the code here
+
+function fetchAndBuildEpisodes(tvShowNumber) {
```

Comment:
Thos fetching function is written very well, it's easy to read and the functionality is very well written.